### DISCUSSION TITLE
Introduce yourself GSOC 2024

### DISCUSSION THREAD

**Discussion by NileshVishwakarma982** [ruxailab/RUXAILAB#398] [General]
Greetings GSoC Community,

I'm Nilesh Vishwakarma, eager to join GSoC 2024. Despite being a latecomer, my enthusiasm for open-source development knows no bounds.

Hailing from Mumbai, India, my passion lies in AI and Data Science. I've explored NLP, data extraction, and sentiment analysis, particularly analyzing IMDb reviews. Contributing to projects like WebDevOpen has sharpened my collaborative skills.

I'm particularly interested in the project: Data Extraction for Sentiment Analysis from Usability Tests. I'm here to expand my knowledge, collaborate, and make a meaningful impact. Excited to learn, grow, and contribute to open-source initiatives alongside you all.

You can find my resume [here](https://drive.google.com/file/d/1euiahJFuuwMncB1eXi4Vx1H3WgK8IVzC/view?usp=drive_link).

Warm regards,
Nilesh Vishwakarma

**Comment by KarinePistili** [comment]
Please use discussion #306  for introductions
